inspectors/view/qml/notation/brackets/internal/BracketPopupSection.qml

```diff
@@ -0,0 +1,37 @@
+import QtQuick 2.9
+import MuseScore.Inspectors 3.3
+
+import "../../../common"
+
+Column {
+
+    property QtObject intBracketProperty: undefined
+    property string labelProperty: undefined
```

Comment:
it's better to initialize string by empty string 
But actually you can just use an alias property
`property alias title: textLabel.text`